vendor/github.com/gardener/etcd-druid/api/v1alpha1/etcd_types.go

```diff
@@ -217,8 +217,13 @@ type EtcdConfig struct {
 	// More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
 	// +optional
 	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
+	// ClientUrlTLS contains the ca, server TLS and client TLS secrets for client communication to ETCD cluster
 	// +optional
-	TLS *TLSConfig `json:"tls,omitempty"`
```

Comment:
This doesn't seem to be a backwards-compatible API change. Will this cause issues?

Reply:
Not for gardener, since we have taken care of migrating to the new API with the new druid release in this PR. We have also added a breaking change release note along with this change.